src/components/accessories/appHeader/AppHeader.tsx

```diff
@@ -113,14 +113,8 @@ const AppHeader: FunctionComponent<TProps> = ({
           </div>
           <div className="appHeader__nav">
             <div className="appHeader__nav_items">
-              <div className="appHeader__nav__item">{t("nav.pharmacy")}</div>
-              <div className="appHeader__nav__item">{t("nav.ward")}</div>
-              <div
-                className="appHeader__nav__item"
-                onClick={() => history.push("/billing")}
-              >
-                {t("nav.billing")}
-              </div>
+              <div className="appHeader__nav__item">{t("nav.visits")}</div>
+              <div className="appHeader__nav__item">{t("nav.laboratory")}</div>
```

Comment:
Maybe we should add here a menu item for **Patients** as well, that would take the user to the main dashboard (the screen with the two buttons "**New Patient**" and "**Search Patient**"). Otherwise one wouldn't be able to return to that screen without changing the URL directly. 

In short, the **AppHeader** menu should have three items: **Patients**, **Visits**, **Laboratory**.